# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/streams/readable-streams/floating-point-total-queue-size.any.js.ini
# Commit: c34fddbfd51b
# Full Hash: c34fddbfd51b86477ee9637f291228d8e6c5f939
# Author: Alexandra Borovova <aborovova@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - Mark all tests running in shadowrealm-in-serviceworker as intermittently failing or timing out.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D230557
# ==============================================================================

diff -r 2afdda37f63b -r c34fddbfd51b testing/web-platform/meta/streams/readable-streams/floating-point-total-queue-size.any.js.ini
--- a/testing/web-platform/meta/streams/readable-streams/floating-point-total-queue-size.any.js.ini	Thu Nov 28 15:09:12 2024 +0000
+++ b/testing/web-platform/meta/streams/readable-streams/floating-point-total-queue-size.any.js.ini	Thu Nov 28 16:46:27 2024 +0000
@@ -25,10 +25,7 @@
   expected: ERROR
 
 [floating-point-total-queue-size.https.any.shadowrealm-in-serviceworker.html]
-  expected:
-    if (processor == "x86_64") and (os == "win") and not debug: ERROR
-    if processor == "x86": ERROR
-    [ERROR, TIMEOUT]
+  expected: [ERROR, TIMEOUT]
 
 [floating-point-total-queue-size.any.shadowrealm-in-sharedworker.html]
   expected: ERROR